config/routes.rb

```diff
@@ -7,7 +7,7 @@
 
   # Accounts
   match '/login' => 'sessions#new', :as => :login, :via => :get
-  match '/login' => 'sessions#create', :as => :login, :via => :post
+  match '/login' => 'sessions#create', :via => :post
```

Comment:
This doesn't cause any existing tests to fail and clicking the login button still works for me; I'm pretty sure we don't need the `POST` route to be named, at least not separately from the `GET` route?